src/otx/algo/classification/dino_v2.py

```diff
@@ -75,6 +77,9 @@ def forward(self, imgs: torch.Tensor, labels: torch.Tensor | None = None, **kwar
             return self.loss(logits, labels)
         return self.softmax(logits)
 
+    def __reduce__(self):
```

Comment:
It depends on class implementation. The reason why dino_v2 model is problematic is that it tries to get model using `torch.hub.load` whose class can't be found w/o any reference. If user want to implement a model in the similar way, then it may be necessary.